src/components/MapVizzard/MapOptions/ColorPicker.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import reactCSS from "reactcss";
+import { SketchPicker } from "react-color";
+import styles from "./ColorPicker.module.css"
+
+export const ColorPicker = ({color, setColor}) => {
+  const [displayColorPicker, setDisplayColorPicker] = useState(false);
+
+  const handleClick = () => {
+    setDisplayColorPicker(!displayColorPicker);
+  };
+
+  const handleClose = () => {
+    setDisplayColorPicker(false);
+  };
+
+  const handleChange = (color) => {
+    setColor(color.rgb);
+  };
+
+
+  return (
+    <div>
+      <div className={styles.swatch} onClick={handleClick}>
+        <div className={styles.color} style={{background: `rgba(${color.r}, ${color.g}, ${color.b}, ${color.a})`}}/>
+      </div>
+      {displayColorPicker ? (
+        <div className={styles.popover}>
+          <div className={styles.cover} onClick={handleClose} />
+          <SketchPicker color={color} onChange={handleChange} />
+        </div>
+      ) : null}
+    </div>
+  );
+};
+
+export default ColorPicker;
+
+export function rgba(rgb) {
+  if (rgb && typeof rgb !== "undefined" && rgb.a > 0) {
+    return "rgba(" + rgb.r + "," + rgb.g + "," + rgb.b + ",+" + rgb.a + ")";
```

Comment:
You can also use Template Literals for this usecase.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals